src/common/file_windows.c

```diff
@@ -68,7 +68,11 @@ util_tmpfile(const char *dir, const char *templ)
 	int oerrno;
 	int fd = -1;
 
-	char *fullname = alloca(strlen(dir) + strlen(templ) + 1);
+	char *fullname = malloc(strlen(dir) + strlen(templ) + 1);
```

Comment:
Please use `Malloc()`/`Free()` (capitalized) to manage memory inside of the library. They are here so that the user interpose custom allocation function.